docs/Changelog.md

```diff
@@ -4,6 +4,112 @@ All notable changes to this project will be documented in this file.
+* Auto detect DynamicDataSourceType by @Youssef1313 in [#4340](https://github.com/microsoft/testfx/pull/4340)
+* Open source Retry and Hot Reload extensions by @Evangelink in [#4354](https://github.com/microsoft/testfx/pull/4354)
+* Add DisplayMessage to TestContext by @Youssef1313 in [#4345](https://github.com/microsoft/testfx/pull/4345)
+* Allow any IEnumerable as return type to DynamicData source member by @Evangelink in [#4389](https://github.com/microsoft/testfx/pull/4389)
+* Allow AssemblyCleanup/ClassCleanup to have TestContext parameter by @Youssef1313 in [#4387](https://github.com/microsoft/testfx/pull/4387)
+* Update ExpectedException codefix to use the new Assert.Throws\[Exactly\]\[Async\] overloads by @Youssef1313 in [#4504](https://github.com/microsoft/testfx/pull/4504)
+* Introduce interpolated string handler overloads for assertions by @Youssef1313 in [#4476](https://github.com/microsoft/testfx/pull/4476)
+* Add analyzer for value type usage with Assert.AreSame by @Youssef1313 in [#4493](https://github.com/microsoft/testfx/pull/4493)
+* Allow DynamicData source to be on base types by @Youssef1313 in [#4482](https://github.com/microsoft/testfx/pull/4482)
+* Use different message for Assert.AreEqual when string difference is casing only by @Youssef1313 in [#4525](https://github.com/microsoft/testfx/pull/4525)
+* Report diagnostic when AreNotSame is passed value type by @Youssef1313 in [#4523](https://github.com/microsoft/testfx/pull/4523)
+* Implement analyzer and codefix to switch to Assert.ThrowsExactly\[Async\] by @Youssef1313 in [#4459](https://github.com/microsoft/testfx/pull/4459)
+* Update ExpectedException analyzer message/description to use the new Assert.Throws methods (and fix typo) by @Youssef1313 in [#4527](https://github.com/microsoft/testfx/pull/4527)
+* Add ability to ignore specific test cases by @Youssef1313 in [#4457](https://github.com/microsoft/testfx/pull/4457)
+* Implement codefix for MSTEST0026: Avoid conditional access in assertions (and small analyzer false positive) by @Youssef1313 in [#4502](https://github.com/microsoft/testfx/pull/4502)
+* Add messageBuilder overloads for Assert.Throws\[Exactly\]\[Async\] by @Youssef1313 in [#4541](https://github.com/microsoft/testfx/pull/4541)
+* Implement codefix for 'Are\[Not\]Same' usage with value types by @Youssef1313 in [#4584](https://github.com/microsoft/testfx/pull/4584)
+* Reword unhandled TestMethodAttribute.Execute exception message by @Evangelink in [#4641](https://github.com/microsoft/testfx/pull/4641)
+* Implement analyzer to warn for use of Assert methods in async void by @Youssef1313 in [#4640](https://github.com/microsoft/testfx/pull/4640)
+* Introduce `RetryAttribute` for test methods by @Youssef1313 in [#4586](https://github.com/microsoft/testfx/pull/4586)
+* Fix bad nullability annotation on CultureInfo by @Evangelink in [#4670](https://github.com/microsoft/testfx/pull/4670)
```

Comment:
Probably belongs to "Fixed" section. @Evangelink?

Reply:
Yes